drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/test/java/org/drools/workbench/screens/scenariosimulation/kogito/client/editor/ScenarioSimulationEditorKogitoWrapperTest.java

```diff
@@ -161,6 +195,58 @@ public void getContent() {
         verify(scenarioSimulationEditorKogitoWrapperSpy, times(1)).transform(eq(scenarioSimulationModelMock));
     }
 
+    @Test
+    public void manageContent() {
+        scenarioSimulationEditorKogitoWrapperSpy.manageContent("path/file.scesim", "value", resolveCallbackFnMock, rejectCallbackFnMock);
+        verify(scenarioSimulationEditorKogitoWrapperSpy, never()).showScenarioSimulationCreationPopup(any());
+        verify(scenarioSimulationEditorKogitoWrapperSpy, times(1)).gotoPath(pathArgumentCaptor.capture());
+        verify(scenarioSimulationEditorKogitoWrapperSpy, times(1)).unmarshallContent(eq("value"));
+        assertEquals("file.scesim", pathArgumentCaptor.getValue().getFileName());
+        assertEquals("path/", pathArgumentCaptor.getValue().toURI());
+    }
+
```

Comment:
Please add a case with empty content and valid path
 scenarioSimulationEditorKogitoWrapperSpy.manageContent("path/file.scesim", "", resolveCallbackFnMock, rejectCallbackFnMock);

Reply:
@dupliaka  Done!